docs/coverage.md

```diff
@@ -0,0 +1,82 @@
+# Coverage
+
+Syzkaller data exported by kcov is used to determine coverage. Kcov is exporting first PC value of each executed code block (see https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/Documentation/dev-tools/kcov.rst). Tools from binutils (objdump, nm, addr2line and readelf) are used to map this data to lines and functions in the source code.
+
+## Binutils
+
+Note that if you are fuzzing in cross-arch environment you need to provide correct Binutils cross-tools to Syzkaller before starting Syz-manager:
```

Comment:
Syzkaller => syzkaller, Syz-manager => `syz-manager`, etc. here and below.

Reply:
Fixed